CDP4SiteDirectory/Views/Dialogs/ParticipantDialog.xaml

```diff
@@ -98,7 +100,9 @@
                         </Button>
                     </Grid>
                 </lc:LayoutItem>
-
+                <items:WarningLayoutItem Visibility="{Binding Path=IsSelectedRoleDeprecated, UpdateSourceTrigger=PropertyChanged,
+                    Converter={StaticResource BooleanToVisibilityConverter }}"
+                                         WarningText="The selected role is deprecated."/>
```

Comment:
We are used to use the 10-25 terminology in user text. So here it would be "ParticipantRole"instead of "role".